ethcore/native_contracts/build.rs

```diff
@@ -43,10 +45,16 @@ fn build_file(name: &str, abi: &str, filename: &str) {
 	f.write_all(code.as_bytes()).unwrap();
 }
 
+fn build_test_contracts() {
```

Comment:
A feature would have to be added since `cfg(test)` isn't propagated down to dependencies.